Added ability to view per-user graphs

otgerp opened the pull request:
Added a `<select>` box at the top of the Graphs page, which allows selecting a single user to see only their data in the graphs. By default it has "All users" selected, which works as before showing the overall statistics.

Implements the feature requested in issue #382

JonnyWong16 commented:
I finally got around to testing this. It looks great!

Can you also filter the "Play count by top 10 users" and "Play count by user and stream type" to the selected user? I know it will only show one bar, but it will be consistent with the other graphs.

JonnyWong16 commented on plexpy/users.py:
```diff
@@ -21,6 +21,23 @@ class Users(object):
+
+            result = monitor_db.select(query)
+        except Exception as e:
+            logger.warn(u"PlexPy Graphs :: Unable to execute database query for get_user_names: %s." % e)
```
Typo:
```
PlexPy Users :: Unable to execute database query for get_names
```

otgerp commented:
Done! I added filtering to those two graphs too.

As for the "typo", actually other functions below that one I added in "users.py" also had mismatching names/error messages. I think the messages correspond to the API call names at "webserve.py". Anyway, I changed the function name so that both are the same.

JonnyWong16 commented:
Thanks, looks good.

Merging so I can use your updated SQL queries in #677.
